static/js/types/storage.d.ts

```diff
@@ -0,0 +1,12 @@
+export interface LxdStorage {
+  config: {
+    "volume.block.filesystem": string;
+    "volume.size": string;
+  };
+  description: string;
+  driver: string;
+  locations: string[];
+  name: string;
+  status: string;
+  used_by?: string[];
+}
```

Comment:
Sorry for the nitpick, there should be an empty line here.

I wonder why the linter doesn't catch this atm.

Reply:
@edlerd resolved